Matey/Extensions/UIViewController + Ext.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIViewController + Ext.swift
+//  Matey
+//
+//  Created by Firat Polat on 30.08.2022.
+//
+
+import Foundation
+import UIKit
+
+extension UIViewController {
+
+    //alert controller
+    func showAlert(title:String?, message:String?, buttonName:String = "Done", completion: (()->())? = nil) {
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        let doneAction = UIAlertAction(title: buttonName, style: UIAlertAction.Style.default) { (action) in
```

Comment:
```suggestion
        let doneAction = UIAlertAction(title: buttonName, style: UIAlertAction.Style.default) { _ in
```
action is not used why is it there?